style.scss

```diff
@@ -460,6 +460,27 @@ article {
     margin-right: 8px;
     border-right: 1px solid $lightGray;
   }
+
+  .img-wrapper {
```

Comment:
Looking at this, it seems like the SVG might explode to cover the whole page. I assume you have tested it :)

Reply:
Added a preview